packages/@okta/vuepress-site/docs/api/getting_started/rate-limits/index.md

```diff
@@ -118,6 +118,7 @@ The following are the high capacity rate limits per minute that apply across the
 | `/app/{app}/{key}/sso/saml`                                                | 1500      | 3000         |
 | `/api/v1/groups/{id}`                                                      | 1500      | 3000         |
 | `/api/v1/users/{id}`                                                       | 1500      | 3000         |
+| `/api/v1/users/{idOrLogin}` (only `GET`)                                   | 1500      | 5000         |
```

Comment:
why do we put `idOrLogin` here but only `id` in the WRITE version of the endpoint?

Reply:
That is how it is defined in developer docs. I want to keep it that way

<img width="755" alt="Screen Shot 2019-05-30 at 12 29 06 PM" src="https://user-images.githubusercontent.com/19719701/58658923-8bb4b980-82d6-11e9-9582-40875089289a.png">